string name1
float64 left_vtarget
string name2
float64 right_vtarget